conf/app.ini

```diff
@@ -20,6 +20,26 @@ MAX_CREATION_LIMIT = -1
 ; Patch test queue length, make it as large as possible
 PULL_REQUEST_QUEUE_LENGTH = 10000
 
+[editor]
+; List of file extensions that should have line wraps in the CodeMirror editor
+; Separate extensions with a comma. To line wrap files w/o extension, just put a comma
+LINE_WRAP_EXTENSIONS = .txt,.md,.markdown,.mdown,.mkd,,
+; Valid file modes that have a preview API associated with them, such as api/v1/markdown
+; Separate values by commas. Preview tab in edit mode won't show if the file extension doesn't match
+PREVIEW_TAB_APIS = markdown
+
+[upload]
```

Comment:
Not sure what you mean here. [attachment] is actually just for issues, and I didn't touch that, so that can be set differently than repository uploads. For example, in a repo, you may want allow any file type to be upload, but for Issues you may just want PDFs and images. If [attachment] should be changed to [issues] that should be a separate pull request.